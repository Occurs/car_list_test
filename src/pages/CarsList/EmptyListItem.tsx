import React from 'react';
import Box from '@material-ui/core/Box';
import { colors } from 'styles/variables';

const EmptyListItem = () => {
  return (
    <Box display='flex' border={`1px solid ${colors.secondary}`} padding='12px' height='90px' marginTop='12px' width='100%'>
      <Box component='div' bgcolor={colors.secondary} marginRight='12px' height='100%' width='85px' />
      <Box component='div' marginLeft='12px' height='100%' display='flex' alignItems='flex-start' flexDirection='column' justifyContent='space-between'>
        <Box component='div' height='18px' width='180px' bgcolor={colors.secondary} />
        <Box component='div' height='14px' width='180px' bgcolor={colors.secondary} />
        <Box component='div' height='14px' width='60px' bgcolor={colors.secondary} />
      </Box>
    </Box>
  );
};

export default EmptyListItem;
