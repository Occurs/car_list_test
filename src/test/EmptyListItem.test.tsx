
import React from "react";
import renderer from "react-test-renderer";
import EmptyListItem from "pages/CarsList/EmptyListItem";

describe("Footer Component", () => {
  test("snapshot", () => {
    const component = renderer.create(<EmptyListItem />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
