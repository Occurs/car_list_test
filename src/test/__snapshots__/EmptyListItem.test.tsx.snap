// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiBox-root MuiBox-root-2"
  />
  <div
    className="MuiBox-root MuiBox-root-3"
  >
    <div
      className="MuiBox-root MuiBox-root-4"
    />
    <div
      className="MuiBox-root MuiBox-root-5"
    />
    <div
      className="MuiBox-root MuiBox-root-6"
    />
  </div>
</div>
`;
